libs/domain/picking/src/components/filters/filters.model.ts

```diff
@@ -0,0 +1,33 @@
+import { FormFieldDefinition, FormFieldType } from '@spryker-oryx/form';
+import { i18n } from '@spryker-oryx/utilities';
+
+export interface FiltersAttributes {
+  open?: boolean;
+}
+
+export const fields: FormFieldDefinition[] = [
+  {
+    id: 'sortBy',
+    type: FormFieldType.RadioList,
+    label: i18n('picking.filter.sort-by') as string,
+    attributes: { direction: 'vertical' },
+    options: [
+      {
+        value: 'requestedDeliveryDate.desc',
+        text: 'Earliest pick-up time',
```

Comment:
These texts should also be translated by i18n.

Reply:
yeah, i'd make those text a i18n key